processor/processor.go

```diff
@@ -35,6 +35,13 @@ import (
 	"errors"
 	"expvar"
 	"fmt"
+	"image"
+
+	// Register file-types for image size extraction
+	_ "image/gif"
```

Comment:
Go complains about unused imports, `_` allows us to import a package that is not used.

Why would you want to do that? Because of the import side-effects. When a package is imported its `init()` function is called, in that case `init()` registers the mime-type.

See https://github.com/golang/go/blob/0e85fd7561de869add933801c531bf25dee9561c/src/image/gif/reader.go#L636